TestBenches/TrackletProcessor_test.cpp

```diff
@@ -28,13 +28,13 @@
   #define SEED_ L1L2_
 #endif
 #if !defined ITC_
-  #define ITC_ D_
+  #define ITC_ C_
 #endif
 #if !defined MODULE_
-  #define MODULE_ TP_L1L2D_
+  #define MODULE_ TP_L1L2C_
 #endif
 #if !defined TOP_FUNC_
-  #define TOP_FUNC_ TrackletProcessor_L1L2D
+  #define TOP_FUNC_ TrackletProcessor_L1L2C
 #endif
```

Comment:
@aryd These macros should correspond to this module here:
https://github.com/cms-L1TK/firmware-hls/blob/a3e5decfe1ed903ab9c66c948ba16e3811c6fc68/project/script_TP.tcl#L15-L17
Otherwise, the C/RTL cosimulation will fail. It doesn't fail right now because the `module_to_export` is not in the list of `modules_to_test` here:
https://github.com/cms-L1TK/firmware-hls/blob/a3e5decfe1ed903ab9c66c948ba16e3811c6fc68/project/script_TP.tcl#L9-L14

Can you set the `module_to_export` in script_TP.tcl to `TP_L1L2C` to match these macros and so that the cosimulation runs?

Reply:
Fixed.